shared/js/abp-preprocessed.es6.js

```diff
@@ -153,3 +157,11 @@ function getVersionParam () {
 
     return versionParam
 }
+
+window.abpLists = (() => {
+    return {
+        getWhitelists,
+        getEasylists,
+        updateLists
+    }
+})()
```

Comment:
Because this file is currently one of two files that get browserified, all variables that get defined with `var`, `let` or `const` don't leak onto the global scope. (Compare that with e.g. `trackers.js` where all variable definitions at the top end up on the global scope.)

The other modules currently use a weird definition of `require` which is not compatible with Browserify, so I'm adding this on the `window` scope temporarily to make this into a fake module of sorts. Once we implement browserify, I'll change `window.abpLists` to `module.exports` and all will be well.